ca/server.go

```diff
@@ -740,11 +737,13 @@ func (s *Server) signNodeCert(ctx context.Context, node *api.Node) error {
 	)
 
 	// Try using the external CA first.
-	cert, err := externalCA.Sign(ctx, PrepareCSR(rawCSR, cn, ou, org))
+	s.secConfigMu.Lock()
```

Comment:
Would a read lock be better?  Alternately, we can not lock at all for now - a node will normally re-try the renew anyway.  In one of my future PRs, the server has its own external CA and root CA that will just be replaced in the `UpdateRootCA` code, and we can lock getting those instead of locking during signing?